modules/lsm/lsm_module_iface.c

```diff
@@ -166,6 +177,41 @@ _drive_update (UDisksObject   *object,
     }
 }
 
+static gboolean
+_lsm_local_check (UDisksObject *object)
+{
+  /* LsmLocalDisk interface will be available on all disk drive */
+  return TRUE;
```

Comment:
Isn't this run on every object or at least on every block device? I don't think e.g. partitions should have this interface, or should they?